### PR TITLE
add error message to all `nnictl package xxx`

### PR DESCRIPTION
**Pull request by J-shang**

### CODE REVIEW

**Review by QuanluZhang — APPROVED** [review]

**Review by SparkSnail — APPROVED** [review]

File: nni/tools/nnictl/nnictl.py [hunk 1]
```diff
@@ -243,12 +243,9 @@ def parse_args():
     def show_messsage_for_nnictl_package(args):
         print_error('nnictl package command is replaced by nnictl algo, please run nnictl algo -h to show the usage')
 
-    parser_package_subparsers = subparsers.add_parser('package', help='control nni tuner and assessor packages').add_subparsers()
-    parser_package_subparsers.add_parser('install', help='install packages').set_defaults(func=show_messsage_for_nnictl_package)
```
Comment (SparkSnail) [inline nni/tools/nnictl/nnictl.py:247]: Does these subcommand still supported? What is the output in `nnictl package --help` command after remove these parser here?

Comment (J-shang) [reply nni/tools/nnictl/nnictl.py:247]: all subcommand don't support, `--help` will show
```
usage: nnictl package [-h] ...

positional arguments:
  args

optional arguments:
  -h, --help  show this help message and exit
```
maybe `add_help=False` is better?

Comment (SparkSnail) [reply nni/tools/nnictl/nnictl.py:247]: Can you add information to explain `args` here? like `parser_package_subparsers.add_argument('args', help='explain args', nargs=argparse.PEMAINDER)`.

Comment (J-shang) [reply nni/tools/nnictl/nnictl.py:247]: fix it
```
usage: nnictl package [-h] ...

positional arguments:
  args        all subcommands have deprecated, nnictl package command is
              replaced by nnictl algo, please run nnictl algo -h to show the
              usage

optional arguments:
  -h, --help  show this help message and exit
```

**Pull request merged**